features/dd-sdk-android-session-replay-material/src/main/kotlin/com/datadog/android/sessionreplay/material/MaterialExtensionSupport.kt

```diff
@@ -29,7 +30,7 @@ class MaterialExtensionSupport : ExtensionSupport {
     private val viewIdentifierResolver: ViewIdentifierResolver = DefaultViewIdentifierResolver
     private val colorStringFormatter: ColorStringFormatter = DefaultColorStringFormatter
     private val viewBoundsResolver: ViewBoundsResolver = DefaultViewBoundsResolver
-    private val drawableToColorMapper: DrawableToColorMapper = DrawableToColorMapper.getDefault()
+    private val drawableToColorMapper: DrawableToColorMapper = DrawableToColorMapper.getDefault(InternalLogger.UNBOUND)
```

Comment:
It means no telemetry will be sent for this one, because `InternalLogger.UNBOUND` is not bound to any SDK instance (and hence to the RUM feature). Is it supposed to be like that?

Reply:
Good point thanks